src/screens/HomeScreen.js

```diff
@@ -323,7 +323,7 @@ export const HomeScreen = ({ navigation }) => {
                             params: {
                               title: 'Veranstaltungen',
                               query: 'eventRecords',
-                              queryVariables: { order: 'listDate_ASC' },
+                              queryVariables: { limit: 15, order: 'listDate_ASC' },
                               rootRouteName: 'EventRecords'
                             }
                           })
```

Comment:
limit of 15 queryVariables per time

Reply:
no. `limit` is a query variable with the value 15.

Reply:
[queries-limit](https://www.postgresql.org/docs/8.2/queries-limit.html ) not helping me ... 
why do you set 15 as a value for the query variable limit ?

Reply:
From your link:

> If a limit count is given, no more than that many rows will be returned

We want to request max. 15 items with every request.